src/http/ldp/PostOperationHandler.ts

```diff
@@ -35,7 +37,11 @@ export class PostOperationHandler extends OperationHandler {
       this.logger.warn('POST requests require the Content-Type header to be set');
       throw new BadRequestHttpError('POST requests require the Content-Type header to be set');
     }
-    const identifier = await this.store.addResource(operation.target, operation.body, operation.conditions);
-    return new CreatedResponseDescription(identifier);
+    const result = await this.store.addResource(operation.target, operation.body, operation.conditions);
+    const createdIdentifier = Object.entries(result).find(
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      ([ key, value ]): boolean => value.get(SOLID_AS.terms.Activity)?.value === AS.Create,
+    )![0];
```

Comment:
```suggestion
    const createdIdentifier = Object.entries(result).find(
      ([ , value ]): boolean => value.get(SOLID_AS.terms.Activity)?.value === AS.Create,
    )![0];
```

I'm wondering in the future if there are cases where multiple resources can be created during a POST where this would break. Might have to add extra metadata then. But not really relevant right now.